lib/forkcms_deploy/forkcms_3.rb

```diff
@@ -240,4 +244,14 @@
 			run "mysql --default-character-set='utf8' --host=#{yaml['parameters']['database.host']} --port=#{yaml['parameters']['database.port']} --user=#{yaml['parameters']['database.user']} --password=#{yaml['parameters']['database.password']} #{yaml['parameters']['database.name']} < #{mysql_update_file}"
 		end
 	end
+
+	namespace :opcode do
+		desc "Clears the opcode cache"
+		task :clear do
+			run "touch #{document_root}/reset.php"
```

Comment:
Ik zou hier eerder een random naam genereren. Kwestie dat als /reset.php al bestaat je niet in de shit komt.